app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb

```diff
@@ -87,6 +112,98 @@ def get_templates(inventory)
         end
       end
 
+      ## InventoryObject refresh methods
+
+      def merge_projects_into_namespaces_graph(inventory)
+        collection = @inv_collections[:container_projects]
+
+        inventory["project"].each do |data|
+          h = parse_project(data)
+          # TODO: Assumes full refresh, and running after get_namespaces_graph.
+          #   Would be a problem with partial refresh.
```

Comment:
Must run after get_namespaces_graph` seems like a fair assumption (we have lots of those)

Can you rephrase the TODO's as assertions? 
(I mean something like `# Must run after get_namespaces_graph`, `# need modification for partial refresh`, etc)
I'm not sure there is actually something todo here right now, so let's bring in as little `TODO` as possible.